sources/packages/api/src/route-controllers/institution-locations/institution-location.institutions.controller.ts

```diff
@@ -112,27 +120,46 @@ export class InstitutionLocationInstitutionsController extends BaseController {
    * Get all active application of a location in an institution
    * with application_status is completed.
    * @param locationId location id.
+   * @param searchCriteria Search text to search active applications.
+   * @param sortField Field to sort active applications.
+   * @param page Current page of paginated result.
+   * @param pageLimit Records per page in a paginated result.
+   * @param sortOrder Sort order of active applications.
+   * @param applicationStatus View status of applications requested by user.
    * @returns Student active application list of an institution location.
    */
   @HasLocationAccess("locationId")
   @Get(":locationId/active-applications")
   async getActiveApplications(
     @Param("locationId") locationId: number,
-  ): Promise<ActiveApplicationSummaryAPIOutDTO[]> {
+    @Query() pagination: ApplicationPaginationOptionsAPIInDTO,
```

Comment:
Apparently the `ApplicationPaginationOptionsAPIInDTO` is reused from another endpoint and it allows sorting by `status` and `applicationNumber`, which does not seem to be applicable for this one. Either way, I would recommend creating an `ApplicationPaginationOptionsAPIInDTO` specific for this endpoint.

![image](https://user-images.githubusercontent.com/61259237/174350807-24a50437-dc54-4234-9b63-e0e9bc9ba350.png)